models/booking.js

```diff
@@ -13,21 +13,17 @@ booking.init(
       autoIncrement: true,
     },
 
-    booking_time: {
-      type: DataTypes.STRING,
-      allowNull: false,
-    },
-
-    booking_type: {
-      type: DataTypes.STRING,
-      allowNull: false,
+    //cancelled
+    cancelled: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
     },
 
```

Comment:
Please add a confirmed field with a boolean value to track if the poster has approved the booking

Reply:
done